### ISSUE TITLE
ignore me i am no bug

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10405]
|<img alt="enki's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| enki reported|
|-|-|
|Trac ID|trac#517|
|Type|defect|
|Created|2004-02-20 18:09:51Z|

<details><summary>Searchable metadata</summary>

```
trac-id__517 517
type__defect defect
reporter__enki enki
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1077300591000000 1077300591000000
changetime__1471820961861987 1471820961861987
version__ 
owner__ 
cc__glyph cc__enki
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="enki's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|enki commented|
|-|-|

```
#!html
<pre>
&lt;BODY BGCOLOR=" link="#000000" vlink="#000000" alink="#000000">
  -> missing close quote

&lt;td class=zeile5>Active Translator Kroatisch&lt;=>Deutsch&lt;/td>
   -> this is yuck, but it is there and browsers deal with it.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="enki's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|enki commented|
|-|-|

```
#!html
<pre>
triple submission - twistedmatrix.com was lagging
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Mark bugs as 'resolved' to remove them.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_5"></a>|@rodrigc commented|
|-|-|

In changeset 1698513f582bc65504a33eef9944b64d10d1de0c
```
#!CommitTicketReference repository="" revision="1698513f582bc65504a33eef9944b64d10d1de0c"
Merge pull request #517 from twisted/8778-rodrigc-web-py3

Author: rodrigc
Reviewer: meejah
Fixes: #8778

Fix test_web on Windows with Python 3.5
```
